### DISCUSSION TITLE
uBO and Preventing Context Menu Modifications

### DISCUSSION THREAD

**Discussion by mort666** [uBlockOrigin/uAssets#19140] [Q&A]
Context menus on websites and prventing their modification by the visited site. In my specific instance here on Github. I want to be able to restore the 'Back', 'Forward' and 'Reload' options that are removed and remove the dumb 'Emoji' option they have added to replace the useful menu options because Github in this instance are under the mistaken understanding the site is a Mobile Social Media App and not a web site and source code repository management tool to enable people to actually do useful things. 

Now, I was wondering is there any functionality I may have missed witihn uBO that can prevent sites like Github doing this dumb and frankly invasive practice that alters the browsers core functionality?

I've seen loads of 'how to disable the context menu' and 'extensions to re-enable when it has been disabled' but so far I've not found something that can **restore an altered context menu** which is in this instance the really more useful functionality on this front, as although in my specific use case is more about restoring QoL options, the context menu does provide a way into more privacy and security useful features and restoring those options when they have been removed is again really more useful.

Thanks in advance..

**Comment by MasterKia** [comment]
> restore an altered context menu

Send the exact link of the page you're having this problem with.

**Comment by krystian3w** [comment]
In Firefox press right mouse button with <kbd>Shift</kbd>, this will override many JS blockades (credits by gorhill).

Chromium may not have these feauture.
 Use Safari 12 can be risky even with oldie uBo 1.14.x.
